python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -193,119 +188,17 @@ def getOpSimRecords(self, obsHistID=None, expDate=None, fieldRA=None,
-                               moonRA=None, moonDec=None, rotSkyPos=None, telescopeFilter=None,
-                               rawSeeing=None, seeing=None, sunAlt=None, moonAlt=None, dist2Moon=None,
-                               moonPhase=None, expMJD=None, altitude=None, azimuth=None,
-                               visitExpTime=None, airmass=None, skyBrightness=None,
-                               m5=None, boundType='circle', boundLength=0.1, limit=None):
-
-        """
-        This method will query the OpSim database summary table according to user-specified
-        constraints and return a list of of ObservationMetaData instantiations consistent
-        with those constraints.
-
-        @param [in] limit is an integer denoting the maximum number of ObservationMetaData to
-        be returned
-
-        @param [in] boundType is the boundType of the ObservationMetaData to be returned
-        (see documentation in sims_catalogs_generation/../db/spatialBounds.py for more
-        details)
-
-        @param [in] boundLength is the boundLength of the ObservationMetaData to be
-        returned (in degrees; see documentation in
-        sims_catalogs_generation/../db/spatialBounds.py for more details)
-
-        All other input parameters are constraints to be placed on the SQL query of the
-        opsim output db.  These contraints can either be tuples of the form (min, max)
-        or an exact value the user wants returned.
-
-        Parameters that can be constrained are:
-
-        @param [in] fieldRA in degrees
-        @param [in] fieldDec in degrees
-        @param [in] altitude in degrees
-        @param [in] azimuth in degrees
-
-        @param [in] moonRA in degrees
-        @param [in] moonDec in degrees
-        @param [in] moonAlt in degrees
-        @param [in] moonPhase (a value from 1 to 100 indicating how much of the moon is illuminated)
-        @param [in] dist2Moon the distance between the telescope pointing and the moon in degrees
-
-        @param [in] sunAlt in degrees
-
-        @param [in[ rotSkyPos (the angle of the sky with respect to the camera coordinate system) in degrees
-        @param [in] telescopeFilter a string that is one of u,g,r,i,z,y
-
-        @param [in] airmass
-        @param [in] rawSeeing (this is an idealized seeing at zenith at 500nm in arcseconds)
-        @param [in] seeing (this is the OpSim column 'FWHMeff' or 'finSeeing' [deprecated] in arcseconds)
-
-        @param [in] visitExpTime the exposure time in seconds
-        @param [in] obsHistID the integer used by OpSim to label pointings
-        @param [in] expDate is the date of the exposure (units????)
-        @param [in] expMJD is the MJD of the exposure
-        @param [in] m5 is the five sigma depth of the observation
-        @param [in] skyBrightness
-        """
+        summary_columns = self.opsimdb.get_column_names('Summary')
+        self._set_seeing_column(summary_columns)
```

Comment:
I repeated the ```_set_seeing_column()``` calculation because the new ```ObservationMetaDataFromPointing``` methods mean that users could, in principle, connect to an OpSimDB that using ```finSeeing``` but then pass OpSim records from a newer database that uses ```FWHMeff``` into ```ObservationMetaDataFromPointing()```.

We can probably get away with just setting ```summary_columns``` once, since there are no methods to connect an ```ObservationMetaDataGenerator``` to a new OpSimDB after instantiation.